pyproject.toml

```diff
@@ -28,10 +28,10 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<4.0"
+python = ">=3.10,<4.0"
 ansys-openapi-common = "^2.0.0"
 # Ensure PIP_INDEX_URL is not set in CI when reverting to a public version of the package.
```

Comment:
Probably best to comment out the env variable in CI to ensure we're using the package from public PyPI